resources/run_adf.sh

```diff
@@ -210,7 +221,7 @@ install_mongosh() {
       chmod -R +x $LOCAL_INSTALL_DIR/$MONGOSH_UNZIP_DIR/bin/
       echo $LOCAL_INSTALL_DIR/$MONGOSH_UNZIP_DIR
     else
-      tar zxf $LOCAL_INSTALL_DIR/$MONGOSH_DOWNLOAD_FILE --directory $LOCAL_INSTALL_DIR
+      unzip -qo $LOCAL_INSTALL_DIR/$MONGOSH_DOWNLOAD_FILE -d $LOCAL_INSTALL_DIR 2> /dev/null
```

Comment:
While `tar` works locally, it wasn't working on the build host in evergreen! This file downloads as a `.zip`, so I used `unzip`.

Reply:
That would explain the issue I was seeing!